README.md

```diff
@@ -1,24 +1,106 @@
-# tr1d1um
+# Tr1d1um
 
 [![Build Status](https://travis-ci.org/Comcast/tr1d1um.svg?branch=master)](https://travis-ci.org/Comcast/tr1d1um) 
 [![codecov.io](http://codecov.io/github/Comcast/tr1d1um/coverage.svg?branch=master)](http://codecov.io/github/Comcast/tr1d1um?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/Comcast/tr1d1um)](https://goreportcard.com/report/github.com/Comcast/tr1d1um)
 [![Apache V2 License](http://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/Comcast/tr1d1um/blob/master/LICENSE)
 
-The Webpa micro-service that encode TR-181 requests.
+An implementation of the [WebPA API](https://swagger.webpa.comcast.net) which supports TR-181 data model requests and leverages the
+[XMiDT API]((https://github.com/Comcast/xmidt)) under the hood.
 
-# How to Install
 
-## Centos 6
 
-1. Import the public GPG key (replace `0.0.1-65` with the release you want)
 
+
+## Summary
+The WebPA API operations can be divided into the following categories:
+
+1) **Device Statistics**
+
+- `GET /device/{device_id}/stat`
+
+Fetch the statistics for a given device connected to the XMiDT cluster
+
+2) **CRUD operations against devices connected to the XMiDT cluster**
+
+- `[GET|PATCH] /device/{device_id}/config`
+
+- `[POST|PUT] /device/{device_id}/config/{table}`
+
+- `DELETE /device/{device_id}/config/{row}`
+
+Tr1d1um validates the incoming request, injects it into the payload of a SimpleRequestResponse [WRP](https://github.com/Comcast/wrp-c/wiki/Web-Routing-Protocol) message and and sends it to XMiDT which then routes it to the specified device.
+
+3) **Event listener registration**
+- `[POST/GET] /hook`
+
+## Details 
```

Comment:
The current summary should be moved down to `details` and then we should have a 2-3 sentence summary of what Tr1d1um does.

Reply:
I agree with Jack; what is under Summary looks more like a part of Details.  Usually the summary is just a 2-4 sentence paragraph that explains at a high level what the service does.
https://github.com/xmidt-org/svalinn#summary